_config.yml

```diff
@@ -57,3 +57,5 @@ exclude:
 - vendor/cache/
 - vendor/gems/
 - vendor/ruby/
+- _wetboew-demos
+- méli-mélo-demos
```

Comment:
What was the error related for those 2 folder. Those folder need to be build by Jekyll in order to spin up compiled working example of the méli-mélo and idem for the wet-boew demos rendered with the GCWeb theme.

I do understand there is some link issue where the fix need to be about the link not excluding them.